static-catalog/implementations/DigitalGlobe/examples/DigitalGlobeAcquisition.json

```diff
@@ -0,0 +1,320 @@
+          [
+            -104.9406978,
+            39.65662621
+          ],
+          [
+            -104.9412063,
+            39.60882697
+          ],
+          [
+            -104.94199,
+            39.56114115
+          ],
+          [
+            -104.9429155,
+            39.51371447
+          ],
+          [
+            -104.9433825,
+            39.46682092
+          ],
+          [
+            -104.9436075,
+            39.42030036
+          ],
+          [
+            -104.9441694,
+            39.37382041
+          ],
+          [
+            -104.9450015,
+            39.32748833
+          ],
+          [
+            -104.9454518,
+            39.28152803
+          ],
+          [
+            -104.9459165,
+            39.23581207
+          ],
+          [
+            -104.9476415,
+            39.18976879
+          ],
+          [
+            -104.9485045,
+            39.14430328
+          ],
+          [
+            -104.9491347,
+            39.09911019
+          ],
+          [
+            -104.9500794,
+            39.05402796
+          ],
+          [
+            -104.9507337,
+            39.00920829
+          ],
+          [
+            -104.9506689,
+            38.96469654
+          ]
+        ]
+      ]
+    ]
+  },
+  "properties": {
+    "bands": [
+      {
+        "common_name": "COASTAL",
+        "gsd": 2.047,
+        "center_wavelength": 427.0,
+        "effective_bandwidth": 62.0
+      },
+      {
+        "common_name": "Blue",
+        "gsd": 2.047,
+        "center_wavelength": 478.0,
+        "effective_bandwidth": 73.0
+      },
+      {
+        "common_name": "Green",
+        "gsd": 2.047,
+        "center_wavelength": 546.0,
+        "effective_bandwidth": 80.0
+      },
+      {
+        "common_name": "Yellow",
+        "gsd": 2.047,
+        "center_wavelength": 608.0,
+        "effective_bandwidth": 48.0
+      },
+      {
+        "common_name": "Red",
+        "gsd": 2.047,
+        "center_wavelength": 659.0,
+        "effective_bandwidth": 70.0
+      },
+      {
+        "common_name": "RedEdge",
+        "gsd": 2.047,
+        "center_wavelength": 724.0,
+        "effective_bandwidth": 50.0
+      },
+      {
+        "common_name": "NIR",
+        "gsd": 2.047,
+        "center_wavelength": 833.0,
+        "effective_bandwidth": 136.0
+      },
+      {
+        "common_name": "NIR",
+        "gsd": 2.047,
+        "center_wavelength": 949.0,
+        "effective_bandwidth": 187.0
+      },
+       {
+        "common_name": "PAN",
+        "gsd": 0.512,
+        "center_wavelength": 627.0,
+        "effective_bandwidth": 361.0
+      }
+    ],
+    "startDateTime": "2015-11-09T18:04:44.203Z",
+    "endDateTime": "2015-11-09T18:04:44.203Z",
+    "links": [
+      {
+        "rel": "self",
+        "href": "uri of self"
+      }
+    ],
+    "feature_type": [
+      "GBDXCatalogRecord",
+      "Acquisition",
+      "DigitalGlobeAcquisition",
+      "WV02"
+    ],
```

Comment:
Is the idea in the feature_type that the above feature_type's would all have schemas that fields in this record would validate against? Or is it just meant to be informative in some way?

Reply:
That list is more because that is how our current catalog "tags" the records, so that you can search for and find all WV02 related records (not just acquisitions) for instance.  There is a singular type, DigitialGlobeAcquisition that this record would validate against.